.github/workflows/blackduck.yaml

```diff
@@ -0,0 +1,73 @@
+name: Black Duck Policy Check
+on:
+  schedule:
+    - cron: "0 0 * * *" # Midnight
+  push:
+    branches:
+      - main
+      - 'release-*'
+  pull_request:
+    paths:
+    - '.github/**'
+  pull_request_target:
+    paths-ignore:
+    - '.github/**'
+
+jobs:
+  security:
+    if: github.repository == 'nutanix-cloud-native/cluster-api-provider-nutanix'
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install devbox
+        uses: jetify-com/devbox-install-action@v0.12.0
+        with:
+          enable-cache: true
+
+      - name: Go cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Build Project
+        run: devbox run -- make build
+
+      - name: Set up Java 11
+        uses: actions/setup-java@v4
+        with:
+          java-version: "11"
+          distribution: "adopt"
+
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
```

Comment:
Above, we built CAPX within the devbox environment, and of course that means that environment has go installed. Why do we need this?

Reply:
Good point, I had originally lifted the workflow spec from [CAREN blackduck workflow](https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/.github/workflows/blackduck.yaml). Let me try removing it to see if this still works of if our devbox setup causes some go env pollution preventing a clean run for the tool.